src/main/java/com/infobip/interviewdemo/api/CrudController.java

```diff
@@ -0,0 +1,31 @@
+package com.infobip.interviewdemo.api;
+
+import com.infobip.interviewdemo.domain.CommentEntity;
+import com.infobip.interviewdemo.dto.CommentRequest;
+import com.infobip.interviewdemo.service.CommentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("comment")
+@RequiredArgsConstructor
+public class CrudController {
```

Comment:
```suggestion
public class CommentController {
```
`CrudController` is too generic naming